### DISCUSSION TITLE
[Feature] Android app widget

### DISCUSSION THREAD

**Discussion by kaibagley** [immich-app/immich#10974] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hey,

I thought that a nice addition to the Android app would be a "photo frame" similar to the Aves widget, where it will show a random image that changes periodically. The only problem I have with the Aves widget is that it obviously only pulls from my local images, but I have thousands on my server that I'd love to have resurface randomly!

Thanks for your consideration! And thanks for making one of the most amazing pieces of open source software I've had the pleasure of using!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6082

**Reply by kaibagley** [reply]
Oops, sorry. Was searching the issues...
